modules/@apostrophecms/form-file-field-widget/ui/src/index.js

```diff
@@ -13,10 +13,14 @@ export default () => {
   };
 
   const sizeLimiter = (input) => {
+    if ((input.dataset.maxSize || false) === false) {
+      return;
+    }
+
     const { files } = input;
     const totalSize = Array.from(files || []).reduce((sum, { size }) => sum + size, 0);
 
-    const units = JSON.parse(input.dataset.fileSizeUnits) || {};
+    const units = JSON.parse(input.dataset.fileSizeUnits || '') || {};
```

Comment:
`JSON.parse()` of empty string triggers error in browser.
Maybe:
```javascript
    const units = JSON.parse(input.dataset.fileSizeUnits || '{}');
```

Reply:
ohhh good point